yt_dlp/extractor/sejmpl.py

```diff
@@ -0,0 +1,218 @@
+from .common import InfoExtractor
+from .redge import RedCDNLivxIE
+from ..utils import (
+    clean_html,
+    js_to_json,
+    strip_or_none,
+    traverse_obj,
+    update_url_query,
+)
+
+from datetime import datetime, timedelta
+
+
+def is_dst(date):
+    year = date.year
+    # last sunday of march and october, respectively. might break on switch days.
+    dst_start = datetime(year, 3, 31, 2) - timedelta(days=(datetime(year, 3, 31).weekday() + 1) % 7)
+    dst_end = datetime(year, 10, 31, 3) - timedelta(days=(datetime(year, 10, 31).weekday() + 1) % 7)
+    return dst_start <= date <= dst_end
+
+
+def rfc3339_to_atende(date):
+    date = datetime.fromisoformat(date)
+    date = date + timedelta(hours=1 if is_dst(date) else 0)
+    return int((date.timestamp() - 978307200) * 1000)
```

Comment:
in light of https://github.com/yt-dlp/yt-dlp/pull/8978/commits/f070f2d48e39dd65351307590370941f30cdbe07 we may want to revise this somehow

Reply:
fuck that, just put `datetime.datetime`